components/openapi/src/main/scala/pl/touk/nussknacker/openapi/SwaggerSecurity.scala

```diff
@@ -18,4 +18,12 @@ sealed trait SecurityInHeader extends SwaggerSecurity {
 
 case class ApiKeyInHeader(name: String, key: String) extends SecurityInHeader {
   def value: String = key
+}
```

Comment:
do we need `SecurityInHeader` trait? Is it possible that, there will be different implementations?

Reply:
Authorization header with Bearer token will be such case I think